GoogleDataTransportCCTSupport/GDTCCTWatchOSTestApp/GDTCCTWatchOSTestAppWatchKitExtension/proto/flltest.proto

```diff
@@ -0,0 +1,48 @@
+/*
```

Comment:
I'm not sure we need another copy of the proto in GDTCCT's directory--it's already in there.